> Unrelated: Should the `$build_bitcoin_tx` be dropped from here after #26086?

> Added.

Apparently,  I was wrong. Tested f8b68c1f63f3a942712ef4eeebbdb231b3884f13:
```
$ ./autogen.sh
$ ./configure -q --without-utils --enable-util-tx --without-daemon --disable-tests -disable-bench --disable-fuzz-binary --without-libs CONFIG_SITE=$PWD/depends/x86_64-pc-linux-gnu/share/config.site
$ make
...
  CXX      common/libbitcoin_common_a-interfaces.o
common/interfaces.cpp:8:10: fatal error: boost/signals2/connection.hpp: No such file or directory
    8 | #include <boost/signals2/connection.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
...
```

Sorry for the noise.

--- a/configure.ac
+++ b/configure.ac
@@ -1451,7 +1451,7 @@ if test "$use_natpmp" != "no"; then
   CPPFLAGS="$TEMP_CPPFLAGS"
 fi
 
-if test "$build_bitcoin_wallet$build_bitcoin_cli$build_bitcoin_tx$build_bitcoin_util$build_bitcoind$bitcoin_enable_qt$use_tests$use_bench" = "nononononononono"; then
+if test "$build_bitcoin_wallet$build_bitcoin_cli$build_bitcoin_tx$build_bitcoin_util$build_bitcoind$bitcoin_enable_qt$use_tests$use_bench$enable_fuzz_binary" = "nonononononononono"; then
   use_boost=no
 else
   use_boost=yes

